app/views/godmin/resource/_table.html.erb

```diff
@@ -6,7 +6,7 @@
           <th></th>
         <% end %>
         <% @resource_service.attrs_for_index.each do |attr| %>
-          <th>
+          <th class="column-<%= attr %>">
```

Comment:
Added the class because it was easier to find the right link in the integration test.
Any thoughts on that? Use `qa--` style?